cmd/util/flags/configflags/local_publisher.go

```diff
@@ -0,0 +1,24 @@
+package configflags
+
+import "github.com/bacalhau-project/bacalhau/pkg/config/types"
+
+var LocalPublisherFlags = []Definition{
```

Comment:
does it make sense to allow users to disable this publisher? I would assume we want to disable this our demo network and in marketplace setup as well

Reply:
Yes, can't we use `--disabled-publisher`/`types.NodeDisabledFeaturesPublishers` to do that?

Reply:
I didn't know that!